conf/distro/poky-sota-nonostree.conf

```diff
@@ -0,0 +1,19 @@
+require conf/distro/poky.conf
+
```

Comment:
It looks like almost the same as conf/distro/poky-sota-systemd.conf  but I see that
`require conf/distro/sota.conf.inc` is not present here, was it removed for purpose ?

Reply:
I'm just not sure that it has to inherit from "sota", but I will try! The main idea is that we want to skip to use any ostree related things